toolchains/cross_compiler/cc-toolchain-config.bzl

```diff
@@ -173,10 +174,10 @@ def _impl(ctx):
         toolchain_identifier = ctx.attr.toolchain_identifier,
         host_system_name = "local",
         target_system_name = "arm-frc2023-linux-gnueabi",
```

Comment:
This should probably change to reflect the target triple for the aarch64 targets too, right?

Reply:
Sure, done.  I think I got them right, they are at least close.